mkdocs/themes/mkdocs/base.html

```diff
@@ -114,37 +114,24 @@
                         {%- if 'search' in config.plugins %}
                         <li class="nav-item">
                             <a href="#" class="nav-link" data-bs-toggle="modal" data-bs-target="#mkdocs_search_modal">
-                                <i class="fa fa-search"></i> {% trans %}Search{% endtrans %}
+                                <i class="fa fa-search"></i><span class="d-lg-none ms-2">Search</span>
                             </a>
                         </li>
                         {%- endif %}
                       {%- endblock %}
 
-                      {%- block next_prev %}
-                        {%- if page and (page.next_page or page.previous_page) %}
-                            <li class="nav-item">
-                                <a rel="prev" {% if page.previous_page %}href="{{ page.previous_page.url|url }}" class="nav-link"{% else %}class="nav-link disabled"{% endif %}>
-                                    <i class="fa fa-arrow-left"></i> {% trans %}Previous{% endtrans %}
-                                </a>
-                            </li>
-                            <li class="nav-item">
-                                <a rel="next" {% if page.next_page %}href="{{ page.next_page.url|url }}" class="nav-link"{% else %}class="nav-link disabled"{% endif %}>
-                                    {% trans %}Next{% endtrans %} <i class="fa fa-arrow-right"></i>
-                                </a>
-                            </li>
-                        {%- endif %}
-                      {%- endblock %}
+                      {%- block next_prev %}{%- endblock %}
 
                       {%- block repo %}
                         {%- if page and page.edit_url %}
                             <li class="nav-item">
                                 <a href="{{ page.edit_url }}" class="nav-link">
```

Comment:
```suggestion
                                <a href="{{ repo_url }}" class="nav-link">
```

Reply:
Actually I think that both the `edit_url` and `next_prev` can go the page/content footer. That's how other generators mostly place them by default (and some inhouse boutique docs stacks too) and consumers are likely to be looking for them down there (or, basically acting as a CTA after finishing reading the current page — prev, next, or edit if needs updating).

Good call to have the repo icon in header only link to the `repo_url` to avoid any confusion (edit links render as 404 for logged-out users etc.), again pretty much a pattern nowadays.

Reply:
PS: watch out for the `{%- if page and page.edit_url %}` condition right above if switching away from page.edit_url here…